### PR TITLE
New version v0.6.1

### LINKED ISSUE #13: Publish `generateVanityAddress` to NPM
`Wallet.generateVanityAddress` is currently not available on NPM as it was added after the current latest 0.6.0 release.

Please bump the version and publish, so it becomes available. Thanks!

Workaround in the meantime:

```javascript
    // package.json
    "ethereumjs-wallet": "git://github.com/ethereumjs/ethereumjs-wallet.git#21e7127c15be4adc1fee458c669b0c6cb4f2ffcd",
```

### PR DESCRIPTION
**Pull request by holgerd77**
Closes #13.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) 
+(modification: no type change headlines) and this project adheres to 
+[Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+## [0.6.1] - 2018-02-09
```
Comment (axic) [inline CHANGELOG.md]: Will need to bump this date before merge :)

**Comment by axic** [comment]
Will merge after #27.

**Comment by holgerd77** [comment]
Please wait for https://github.com/ethereumjs/ethereumjs-wallet/pull/37 until merge/release.

**Comment by axic** [comment]
Rebased and added all changes.

**Pull request merged**